bpfd/src/utils.rs

```diff
@@ -83,9 +82,25 @@ pub(crate) async fn set_dir_permissions(directory: &str, mode: u32) {
     }
 }
 
-pub(crate) fn create_bpffs(directory: &str) -> anyhow::Result<()> {
-    debug!("Creating bpffs at {}", directory);
+pub(crate) fn create_bpffs(directory: &str) -> anyhow::Result<(), anyhow::Error> {
+    debug!("Creating bpffs at {directory}");
     let flags = MsFlags::MS_NOSUID | MsFlags::MS_NODEV | MsFlags::MS_NOEXEC | MsFlags::MS_RELATIME;
     mount::<str, str, str, str>(None, directory, Some("bpf"), flags, None)
-        .context("unable to mount bpffs")
+        .map_err(|e| anyhow::anyhow!("unable to create bpffs at {directory}: {e:?}"))
```

Comment:
`.context()` is better here.
See: https://docs.rs/anyhow/latest/anyhow/trait.Context.html

You can change the message though